test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,46 @@
+require "test_helper"
+
+describe RecipesController do
+
+  let (:params) {
+    {
+      query: "chicken",
+      uri: "http://www.edamam.com/ontologies/edamam.owl#recipe_7bf4a371c6884d809682a72808da7dc2"
+    }
+  }
+
+  describe "search" do
+    it "can get search page" do
+      get root_path
+
+      must_respond_with :success
+    end
+  end
+
+  describe "results" do
+    it "can get results page" do
```

Comment:
You should also include a negative test-case for the search results and show actions.